enterprise/frontend/src/embedding-sdk/cli/utils/get-suggested-import-path.unit.spec.ts

```diff
@@ -6,6 +6,40 @@ describe("CLI > getSuggestedImportPath", () => {
     [".", "."],
     ["./src/components", "./components"],
   ])("suggests a reasonable import path", (input, suggestion) => {
-    expect(getSuggestedImportPath(input)).toBe(suggestion);
+    expect(
+      getSuggestedImportPath({
+        isNextJs: false,
+        isUsingSrcDirectory: true,
+        componentPath: input,
+      }),
+    ).toBe(suggestion);
+  });
+
+  it("suggests a reasonable default import path for Next.js without src directory", () => {
+    expect(
+      getSuggestedImportPath({ isNextJs: true, isUsingSrcDirectory: false }),
+    ).toBe("../components/metabase");
+
+    expect(
+      getSuggestedImportPath({
+        isNextJs: true,
+        isUsingSrcDirectory: false,
+        componentPath: "foo/bar",
+      }),
+    ).toBe("../foo/bar");
+  });
+
+  it("suggests a reasonable default import path for Next.js with src directory", () => {
+    expect(
+      getSuggestedImportPath({ isNextJs: true, isUsingSrcDirectory: true }),
+    ).toBe("./metabase");
+
+    expect(
+      getSuggestedImportPath({
+        isNextJs: true,
+        isUsingSrcDirectory: true,
+        componentPath: "foo/bar",
+      }),
+    ).toBe("./bar");
   });
```

Comment:
Note that this will be fixed in https://github.com/metabase/metabase/pull/53473 which adds a much more reasonable import path.